Testscripts/Linux/golang-benchmark.sh

```diff
@@ -46,7 +46,11 @@ Prepare_Test_Dependencies()
         dpkg_configure
     fi
     update_repos
-    install_package make gcc wget libgmp3-dev
+    if [[ "${DISTRO_NAME}" == "centos" ]] || [[ "${DISTRO_NAME}" == "redhat" ]] ; then
```

Comment:
could you please double check for redhat, is the distro name is rhel or redhat?

Reply:
It's rhel. Add the test result against redhat.
```
[LISAv2 Test Results Summary]
Test Run On           : 09/25/2019 03:19:08
ARM Image Under Test  : RedHat : RHEL : 7.5 : Latest
Initial Kernel Version: 3.10.0-862.34.2.el7.x86_64
Final Kernel Version  : 3.10.0-862.34.2.el7.x86_64
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:9

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 GOLANG               PERF-GOLANG-BENCHMARK                                                             PASS                 4.65
```